test/specs/marked/marked.json

```diff
@@ -83,6 +83,18 @@
     "html": "<p>``<em>test`</em></p>",
     "example": 14
   },
+  {
+    "section": "Autolinks",
```

Comment:
Is it a problem if the example numbers collide?

We just had example 15 added in master: https://github.com/markedjs/marked/blob/master/test/specs/marked/marked.json

Reply:
I think the example numbers are just used in the shouldPassButFail array and the test descriptions.

The `marked` tests shouldn't ever fail so maybe we should replace `example` with `description` so they won't collide as often

Reply:
> maybe we should replace example with description so they won't collide as often

I'm fine with that. Let's save that change for a different PR though. This looks good now.